docs/framework/react/quick-start.md

```diff
@@ -3,54 +3,132 @@ id: quick-start
+  fieldComponents: {
+    TextField,
+    NumberField,
+  },
+  formComponents: {
+    SubmitButton,
+  },
+  fieldContext,
+  formContext,
+})
+
+const PeoplePage = () => {
+  const form = useAppForm({
     defaultValues: {
-      fullName: '',
+      username: '',
+      age: 0,
+    },
+    validators: {
+      // Pass a schema or function to validate
+      onChange: z.object({
+        username: z.string(),
+        age: z.number().min(13),
+      }),
     },
-    onSubmit: async ({ value }) => {
+    onSubmit: ({ value }) => {
       // Do something with form data
-      console.log(value)
+      alert(JSON.stringify(value, null, 2))
     },
   })
 
   return (
-    <div>
-        <form
-          onSubmit={(e) => {
-            e.preventDefault();
-            e.stopPropagation();
-            form.handleSubmit();
-          }}
-        >
-          <div>
-            <form.Field
-              name="fullName"
-              children={(field) => (
-                <input
-                  name={field.name}
-                  value={field.state.value}
-                  onBlur={field.handleBlur}
-                  onChange={(e) => field.handleChange(e.target.value)}
-                />
-              )}
-            />
-          </div>
-          <button type="submit">Submit</button>
-        </form>
-    </div>
+    <form
+      onSubmit={(e) => {
+        e.preventDefault()
+        form.handleSubmit()
+      }}
+    >
+      <h1>Personal Information</h1>
+      {/* Components are bound to `form` and `field` to ensure extreme type safety */}
+      <form.AppField
+        name="username"
+        children={(field) => <field.TextField label="Full Name" />}
+      />
+      {/* The "name" property will throw a TypeScript error if typo'd  */}
+      <form.AppField
+        name="age"
+        children={(field) => <field.NumberField label="Age" />}
+      />
+      {/* Our customized hook enables usage of `form.AppField` and `form.AppForm` components */}
+      <form.AppForm>
+        <form.SubmitButton />
+      </form.AppForm>
+    </form>
   )
 }
 
 const rootElement = document.getElementById('root')!
-ReactDOM.createRoot(rootElement).render(<App />)
+ReactDOM.createRoot(rootElement).render(<PeoplePage />)
+```
+
+While we generally suggest using `createFormHook` for reduced boilerplate in the long-run, we also support one-off components and other behaviors using `useForm` and `form.Field`:
+
+```tsx
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import { useForm } from '@tanstack/react-form'
+
+const PeoplePage = () => {
+  const form = useForm({
+    defaultValues: {
+      username: '',
+      age: 0
+    },
+    onSubmit: ({ value }) => {
+      // Do something with form data
+      alert(JSON.stringify(value, null, 2));
+    },
+  });
+
+  return (
+    <form.Field
+      name="age"
+      validators={{
+        // We can choose between form-wide and field-specific validators
```

Comment:
I think this comment is a bit stray here since it doesn't show two different ways of doing things.

Reply:
In comparison to the last example it does - just wanted to clarify for those who were looking between the two confused